controllers/scopeinstance_controller.go

```diff
@@ -104,210 +130,169 @@ func (r *ScopeInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 
 	// create required roleBindings and clusterRoleBindings.
 	if err := r.ensureBindings(ctx, in, st); err != nil {
-		log.Log.Error(err, "in creating Role Bindings")
+		log.Log.V(2).Error(err, "in creating RoleBindings")
```

Comment:
Nit: This could be clusterRoleBindings as well, might as well just log the error.